.travis.yml

```diff
@@ -12,40 +12,31 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-        - TEST_COVERAGE=true
-    - php: 7
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7
+    - php: 7.3
       env:
         - DEPS=latest
         - CS_CHECK=true
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
-      env:
-        - DEPS=latest
         - TEST_COVERAGE=true
         - BENCHMARKS=true
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.3
+    - php: nightly
       env:
         - DEPS=lowest
-    - php: 7.3
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
Should probably be `--ignore-platform-req=php`

Reply:
As discussed on slack - this flag accepts value only on Composer v2.